Sources/InteroperabilityTests/InteroperabilityTestCase.swift

```diff
@@ -66,6 +66,27 @@ public enum InteroperabilityTestCase: String, CaseIterable {
   }
 }
 
+public struct InteroperabilityTestError: Error {
```

Comment:
This isn't public API (because the interop tests aren't a product of the package), so this _can_ just be a `public enum`.

Reply:
Although, we don't use it in this module, so can't we just define it in `interoperability-tests`?

Reply:
Also, can we conform it to `CustomStringConvertible`?